torchmetrics/detection/map.py

```diff
@@ -80,6 +80,22 @@ class COCOMetricResults(BaseMetricResults):
     )
 
 
+def segm_iou(inputs: Tensor, targets: Tensor, smooth: float = 1.0) -> Tensor:
```

Comment:
isn't smooth usually a very small eps like 1e-5 that just prevents division by zero?